bool  left_on_off
bool  right_on_off
bool  back
bool  link1
bool  link2
bool  link3
bool  link4
float32 front_lwheel_target_velocity
float32 front_rwheel_target_velocity
float32 back_lwheel_target_velocity
float32 back_rwheel_target_velocity
